src/form.js

```diff
@@ -33,11 +33,18 @@ export default class SimpleteForm extends HTMLElement {
 		let field = this.searchField;
 		field.setAttribute("autocomplete", "off");
 
+		field.setAttribute("role", "combobox");
+		field.setAttribute("aria-autocomplete", "list");
+		field.setAttribute("aria-expanded", "false");
+		customElements.whenDefined(SUGGESTIONS_TAG).
+			then(() => field.setAttribute("aria-owns", this.suggestions.id));
```

Comment:
Just to be sure: This is absolutely necessary, yes? I'm uncomfortable with establishing this [mutual awareness](https://en.wikipedia.org/wiki/Action_at_a_distance_%28computer_programming%29) (form ↔️ suggestions; cf. `#onSelect` below) - but if it's required, so be it.